src/mamba/py_interface.cpp

```diff
@@ -235,7 +236,8 @@ PYBIND11_MODULE(mamba_api, m)
     py::class_<PrefixData>(m, "PrefixData")
         .def(py::init<const std::string&>())
         .def_readwrite("package_records", &PrefixData::m_package_records)
-        .def("load", &PrefixData::load);
+        .def("load", &PrefixData::load)
+        .def("add_virtual_packages", &PrefixData::add_virtual_packages);
```

Comment:
Not sure about this, my hunch is that something more needs to be done to expose member functions.

Reply:
I think what I did works for member data, but not member functions